.github/workflows/build-docs.yaml

```diff
@@ -0,0 +1,55 @@
+name: Publish Documentation
+
+# on:
+#   push:
+#     branches:
+#       - main 
```

Comment:
I'll change this to run only on PRs. This was just for testing the deployment workflow!